{{ cookiecutter.__project_name_kebab }}/README.md

```diff
@@ -67,15 +67,15 @@ flit install
 Note that this project uses [pre-commit](https://github.com/pre-commit/pre-commit). To set up locally:
 
 ```shell
-# if you don't have it yet, globally
+# install pre-commit
 $ python -m pip install pre-commit
```

Comment:
This is another thing I probably should have caught/discussed in the review of #28 .
Personally I would say `pre-commit` is something that should be installed in your local virtualenv for the project. Then you are not reliant on every single project you interact with that uses pre-commit having to be compatible with a single version if `pre-commit` release a breaking change to their config file format or whatever. Any specific reason why this should be installed globally? Any objections if I change this to a dev dependency in `pyproject.toml` and update the docs?

Reply:
no objection. I have yet to encounter and issue with having pre-commit globally, but running everything in a project venv is a great suggestion